client/landing/stepper/declarative-flow/import-flow.ts

```diff
@@ -122,9 +122,15 @@ const importFlow: Flow = {
 
 					if (
 						depUrl.startsWith( 'http' ) ||
-						[ 'blogroll', 'ghost', 'tumblr', 'livejournal', 'movabletype', 'xanga' ].indexOf(
-							providedDependencies?.platform as ImporterMainPlatform
-						) !== -1
+						[
+							'blogroll',
+							'ghost',
+							'tumblr',
+							'livejournal',
+							'movabletype',
+							'xanga',
+							'substack',
```

Comment:
Nit, but alphabetical order would be:

```

'blogroll',
'ghost',
'livejournal',
'movabletype',
'substack',
'tumblr',
'xanga',
```

Or order somewhat by popularity?

Reply:
Make sense. I'll reorder it alphabetically.